try-it.md

```diff
@@ -535,17 +535,23 @@ Sometimes you may want to look directly at Ironic to debug something.
 The metal3-dev-env repository contains a clouds.yaml file with
 connection settings for Ironic.
 
-metal3-dev-env will install the openstack command line tool on the
-provisioning host as part of setting up the cluster. The openstack tool
-will look for clouds.yaml in the current directory or you can copy it to
-~/.config/openstack/clouds.yaml. Version 3.19.0 or higher is needed to
-interact with Ironic using clouds.yaml.
+Metal3-dev-env will install the unified OpenStack and standalone 
+OpenStack Ironic command-line clients on the provisioning host as 
+part of setting up the cluster. 
+
+Note that currently you can use either unified OpenStack client
+or Ironic client. In this example we are using Ironic client to interact
+with Ironic API.
+
+Please make sure to export
+```CONTAINER_RUNTIME``` environment variable before you execute
+commands.
 
 Example:
 
 ```sh
-[notstack@metal3 metal3-dev-env]$ export OS_CLOUD=metal3
```

Comment:
Do we really no longer need to specify which cloud settings to use?

Reply:
I think no, as far as the user doesn't have another openstack credentials. Because, OS_CLOUD anyways defaults to metal3 in https://github.com/metal3-io/metal3-dev-env/blob/5d8cc0a3c29dfb14bd7638fc12f8b909b1ff9231/openstackclient.sh#L24.